astropy/modeling/parameters.py

```diff
@@ -12,11 +12,13 @@
 import functools
 import numbers
 import operator
+# import warnings
```

Comment:
remove this commented out import?

Reply:
The commented out code appears to be to support the adding a deprecation warning. I don't see any reason not to add the deprecation.

Reply:
Adding a deprecation causes numerous problems in cosmology. It appears there's a `Parameter` class in cosmology which use the modeling Parameter class in some way which I haven't tracked down.

I'm fine with enabling the deprecation (and I think we should) if someone familiar with `cosmology` takes a look and fixes the deprecations which will be raised. Perhaps @nstarman ?

Reply:
`cosmology` classes can be translated to Astropy models, with the cosmology Parameter class translated to a modelling Parameter class (same name, no shared inheritance).